### PR TITLE
Footer UI improved version

### LINKED ISSUE #46: Footer UI enhancement
Need some enhancement with the footer UI.

### PR DESCRIPTION
**Pull request by AnubhavSingh99**
foooter ui improved  
issue #46 

**Footer UI Improvements**
Gradient: Modern violet-to-purple background.
Grid: Responsive layout for all screens.
Typography: Improved fonts for clarity.
GitHub: Animated sliding button.
Hover: Smooth link animations.
Color: Consistent violet-purple theme.
Logo: Subtle hover effect.
image

**Mobile version**
image
image

### CODE REVIEW

**Comment by AryanNanda17** [comment]
UI is pretty. Good work on that. 
Screenshot 2024-10-14 at 9 06 27 PM
remove the white space and newsletter thing. Rest good.

**Comment by AryanNanda17** [comment]
put quick links on the right... connect in the middle and adjust left portion accordingly.

**Comment by AnubhavSingh99** [comment]
i have fixed this 
Screenshot from 2024-10-15 16-13-00

**Comment by AryanNanda17** [comment]
LGTM!
WDYT @ParamThakkar123

**Comment by AryanNanda17** [comment]
I was just looking at it and it looks very big... can you decrease the size of all footer components by 30-40%?

**Comment by ParamThakkar123** [comment]
This looks great man !!

**Comment by ParamThakkar123** [comment]
> I was just looking at it and it looks very big... can you decrease the size of all footer components by 30-40%?

Yes @AnubhavSingh99 you can reduce the size when moving to the mobile screen. For desktop it's fine but for mobile I think the footer itself covers the entire viewport 😅

**Comment by AryanNanda17** [comment]
@ParamThakkar123, do you think the spacing here looks a bit inconsistent? Like its not uniform?
Screenshot 2024-10-15 at 8 54 24 PM

**Comment by AryanNanda17** [comment]
And I think if we reduce size here as well by 20% it would be better, wdyt?

**Comment by AryanNanda17** [comment]
In mobile view... you can remove the lo-fi vibe logo from the middle... and centralize quick links

**Comment by ParamThakkar123** [comment]
> @ParamThakkar123, do you think the spacing here looks a bit inconsistent? Like its not uniform? Screenshot 2024-10-15 at 8 54 24 PM

Yes. This one has some gap in there. Between the image and the actual footer

**Comment by ParamThakkar123** [comment]
> In mobile view... you can remove the lo-fi vibe logo from the middle... and centralize quick links

Yes

**Comment by ParamThakkar123** [comment]
> And I think if we reduce size here as well by 20% it would be better, wdyt?

Yeah. 20% is just fine

**Comment by ParamThakkar123** [comment]
Overall the footer looks really awesome @AnubhavSingh99 . Just make these minor changes, as described by @AryanNanda17 and we are ready to merge

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 </p>
  :notes: Generating original Lofi-Music :notes:
 
-[COC](https://github.com/CommunityOfCoders/Inheritance-2023) Inheritance 2023</h3>
```
Comment (ParamThakkar123) [inline README.md:10]: Why this link was removed ?? This needs to be there I think. It's the place from where this amazing project originated.

**Review by ParamThakkar123 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (AryanNanda17) [reply README.md:10]: bro, I have removed it. He didn't pull the changes or something... that's why it's showing that he did that change.

Comment (ParamThakkar123) [reply README.md:10]: Oh. Okay. No problem. We'll add that later 😄 .

**Comment by AnubhavSingh99** [comment]
image
fixed.

**Comment by ParamThakkar123** [comment]
Yeah looks pretty cool now.

**Review by ParamThakkar123 — APPROVED** [review]

**Comment by anuj846k** [comment]
> image fixed.


In mobile version only
Quick links childs are a little bit more towards right, You can centre align  all the footer elements in the footer

**Comment by AryanNanda17** [comment]
LGTM!

**Comment by ParamThakkar123** [comment]
> > image fixed.
> 
> In mobile version only Quick links childs are a little bit more towards right, You can centre align all the footer elements in the footer

Oh, I didn't notice that ! Thanks for pointing. @AnubhavSingh99 can you please look into that one last thing ?

**Pull request merged**